tests/test_core.py

```diff
@@ -52,41 +52,46 @@ def setUp(self):
         """
         super(AggregationUpdaterTestCase, self).setUp()
         self.agg_modified = now() - timedelta(days=1)
-        course_key = CourseKey.from_string('course-v1:edx+course+test')
+        self.course_key = CourseKey.from_string('course-v1:edx+course+test')
         stubcompat = StubCompat([
-            course_key.make_usage_key('course', 'course'),
-            course_key.make_usage_key('html', 'course-html0'),
-            course_key.make_usage_key('html', 'course-html1'),
-            course_key.make_usage_key('html', 'course-html2'),
-            course_key.make_usage_key('html', 'course-html3'),
-            course_key.make_usage_key('other', 'course-other'),
-            course_key.make_usage_key('hidden', 'course-hidden0'),
-            course_key.make_usage_key('html', 'course-other-html4'),
-            course_key.make_usage_key('hidden', 'course-other-hidden1'),
+            self.course_key.make_usage_key('course', 'course'),
+            self.course_key.make_usage_key('chapter', 'course-chapter'),
+            self.course_key.make_usage_key('html', 'course-chapter-html'),
+            self.course_key.make_usage_key('html', 'course-html0'),
+            self.course_key.make_usage_key('html', 'course-html1'),
+            self.course_key.make_usage_key('html', 'course-html2'),
+            self.course_key.make_usage_key('html', 'course-html3'),
+            self.course_key.make_usage_key('other', 'course-other'),
+            self.course_key.make_usage_key('hidden', 'course-hidden0'),
+            self.course_key.make_usage_key('html', 'course-other-html4'),
+            self.course_key.make_usage_key('hidden', 'course-other-hidden1'),
```

Comment:
Reduce future merge issues.

```suggestion
            course_key.make_usage_key('course', 'course'),
            course_key.make_usage_key('html', 'course-html0'),
            course_key.make_usage_key('html', 'course-html1'),
            course_key.make_usage_key('html', 'course-html2'),
            course_key.make_usage_key('html', 'course-html3'),
            course_key.make_usage_key('other', 'course-other'),
            course_key.make_usage_key('hidden', 'course-hidden0'),
            course_key.make_usage_key('html', 'course-other-html4'),
            course_key.make_usage_key('hidden', 'course-other-hidden1'),
```